xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -1377,7 +1377,8 @@ void CGUIWindowVideoBase::GetGroupedItems(CFileItemList &items)
     VIDEODATABASEDIRECTORY::NODE_TYPE nodeType = CVideoDatabaseDirectory::GetDirectoryChildType(m_strFilterPath);
     if (items.GetContent() == "movies" && params.GetSetId() <= 0 &&
         nodeType == NODE_TYPE_TITLE_MOVIES &&
-       (CSettings::GetInstance().GetBool(CSettings::SETTING_VIDEOLIBRARY_GROUPMOVIESETS) || (StringUtils::EqualsNoCase(group, "sets") && mixed)))
+       (CSettings::GetInstance().GetBool(CSettings::SETTING_VIDEOLIBRARY_GROUPMOVIESETS) && !StringUtils::StartsWithNoCase(m_strFilterPath, "videodb://movies/tags/") || (StringUtils::EqualsNoCase(group, "sets") && mixed)) ||
+	   (CSettings::GetInstance().GetBool(CSettings::SETTING_VIDEOLIBRARY_GROUPMOVIESETSINTAGS) && StringUtils::StartsWithNoCase(m_strFilterPath, "videodb://movies/tags/") || (StringUtils::EqualsNoCase(group, "tags") && mixed)))
```

Comment:
It feels wrong to check the filterpath with a typedout string, is there a constant somwhere I can use?

Also, the last part about group && mixed, I don't know if that is needed, I just added it to replecate the line abbove.